debian/control

```diff
@@ -2,7 +2,11 @@ Source: vncdotool
 Maintainer: Marc Sibson <sibson+vncdotool@gmail.com>
 Section: python
 Priority: optional
-Build-Depends: python-setuptools (>= 0.6b3), debhelper (>= 7), python-support (>= 0.8.4)
+Build-Depends:
+ python-setuptools (>= 0.6b3),
+ debhelper (>= 7),
+ python-support (>= 0.8.4),
+ python-sphinx
```

Comment:
yes, debian/rules will call all the dh-helper tools. For the build target, dh will also call dh_auto_build which will call the Makefile. It would be possible to include the docs in a separate vncdotool-doc package, for instance.
best regards
Alex